src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,118 @@
+import userEvent from '@testing-library/user-event';
```

Comment:
userEvent is imported directly. In modern @testing-library/user-event usage you should call userEvent.setup() to create a user instance. Using the global API may still work for older versions but the recommended pattern is userEvent.setup().

**Sugerencia:**
```typescript
Replace direct calls with `const user = userEvent.setup(); await user.click(element);` Also update tests to await properly and avoid flakiness.
```